Gems/Archive/Code/Include/Archive/ArchiveInterfaceStructs.inl

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+namespace Archive
+{
+    // Implement byte storage multipliers
+    inline namespace literals
+    {
+        constexpr AZ::u64 operator"" _kib(AZ::u64 value)
+        {
+            return value * 1024;
+        }
+
+        constexpr AZ::u64 operator"" _mib(AZ::u64 value)
+        {
+            return value * (1024 * 1024);
```

Comment:
nit: since we are dealing in powers of 2
```suggestion
            return value * (2<<20);
```

Reply:
(2 << 20) is really 2 MiB.